Rakefile

```diff
@@ -1,7 +1,54 @@
+require 'pg'
+
 if ENV['RACK_ENV'] != 'production'
   require 'rspec/core/rake_task'
-  
+
   RSpec::Core::RakeTask.new :spec
-  
+
   task default: [:spec]
 end
+
+task :setup do
```

Comment:
You mentioned in your self-review that you would like to have set up your database tables so that there was a one-to-many relationship between users and peeps.  Based on what you've already done here, its actually a fairly simple change.

I'd suggest you take a look at this to see what this would look like that see if you can research what the appropriate addition to your SQL statement would look like:

http://www.databaseprimer.com/pages/relationship_1tox/

I'm probably also going to run a session on table relationships at some point this week,  so it might be interesting for you to come along to that.